build.gradle

```diff
@@ -29,6 +29,9 @@ repositories {
 		name = "Fabric"
 		url = 'https://maven.fabricmc.net/'
 	}
+	maven {
+		url = 'https://plugins.gradle.org/m2/'
```

Comment:
I assume this isnt needed when being used by mods?

Reply:
just use plugin block

Reply:
No, not here. As this is saying that loom should depend on it. Not for it to be used in loom's buildscript

Reply:
It's not, this is just so that I can pull the plugin into the classpath when developing. The `project.getPluginManager().apply("org.jetbrains.gradle.plugin.idea-ext");` line does everything when actually running gradle (as far as I can see it's the equivalent of `plugins { id 'org.jetbrains.gradle.plugin.idea-ext' version '0.7' }`
Actually, why does this not need me to specify a version... not sure if I should.